src/openforms/analytics_tools/templates/analytics_tools/matomo.html

```diff
@@ -4,7 +4,7 @@
 {% load solo_tags %}
 {% get_solo 'analytics_tools.AnalyticsToolsConfiguration' as analytics_tools_config %}
 
-{% if analytics_tools_config.matomo_enabled %}
+{% if analytics_tools_config.enable_matomo_site_analytics %}
```

Comment:
```suggestion
{% if analytics_tools_config.is_matomo_enabled %}
```
this pattern repeats - we have an explicit check for:

* tool is configured correctly
* tool is actually turned on

now you're testing only for the latter